core/templates/pages/blog-dashboard-page/navbar/blog-dashboard-navbar.component.html

```diff
@@ -0,0 +1,20 @@
+<navbar-breadcrumb>
+  <div class="nav navbar-nav oppia-navbar-breadcrumb">
+    <span class="oppia-navbar-breadcrumb-separator"></span>
+    <span class="pt-3">
+      Blog Dashboard
+    </span>
+  </div>
+</navbar-breadcrumb>
```

Comment:
`navbar-breadcrumb` is a placeholder component and it is not expected to be use like this.
It is used in the base-content component for transclusion

Reply:
Thanks Done.